nginx/README.md

```diff
@@ -0,0 +1,47 @@
+# Support™ NGINX
+
+## Setup Nginx for `Identity-Platform`
+
+Support™ depends on Identity, so **you'll need to perform the**
+[**Nginx setup for identity-platform**](https://github.com/guardian/identity-platform/blob/master/README.md#setup-nginx-for-local-development)
+**first**, before you do anything else.
+
+## Support-specific setup
+
+#### Update your hosts file
+
+Add the following local development domain to your hosts file in `/etc/hosts`:
+
+```
+127.0.0.1   sup.thegulocal.com
```

Comment:
Just the precedent of mem.thegulocal.com and sub.thegulocal.com - but I think you're right, `support.thegulocal.com` is better, I'll update it.